app/controllers/admin/dashboard_controller.rb

```diff
@@ -2,6 +2,9 @@ class Admin::DashboardController < ApplicationController
   before_action :authorize!
 
   def index
-
-  end  
+    @uncontacted_companies = Company.paginate(:page => params[:page],
+                                              :per_page => 5
+                                              ).uncontacted
+                                               .order("updated_at asc")
+  end
```

Comment:
Just discussed and caught that this would be annoying in company list to have to scroll down to the bottom to see ones that were just updated. Since we are just calling this the one time, let's not worry about it and if it keeps coming up we can take a look at it then!